content/pt-br/docs/tasks/configure-pod-container/configure-gmsa.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Configurando GMSA Para Pods e Contêineres Windows
+content_type: task
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.18" state="stable" >}}
+
+Esta página mostra como configurar [Contas de serviço gerenciadas em grupo]
+(https://docs.microsoft.com/en-us/windows-server/security/group-managed-service-accounts/group-managed-service-accounts-overview) (GMSA) 
+para Pods e contêineres que vão executar em nós Windows. Contas de serviço gerenciadas em grupo 
+são um tipo específico de conta do `Active Directory` que provê gerenciamento automático 
+de senhas, gerenciamento simplificado de `service principal name` (SPN), e a habilidade 
+de delegar o gerenciamento a outros administradores através de múltiplos servidores.
+
+No Kubernetes, especificações de credenciais GMSA são configuradas dentro do escopo 
+do cluster Kubernetes como recursos personalizados. Os Pods Windows, assim como contêineres 
+individuais dentro de um Pod, podem ser configurados para usar as funções GMSA 
+baseadas em domínio (ex. autenticação Kerberos) quando interagirem com outros 
+serviços Windows.
+
+## {{% heading "prerequisites" %}}
+
+Você precisa ter um cluster Kubernetes, e a ferramenta de linha de comando `kubectl` 
+precisa estar configurada para comunicar-se com seu cluster.
+O cluster está esperando que haja nós trabalhadores Windows. 
+Esta seção cobre o conjunto inicial de passos requeridos para cada cluster:
+
+### Instale o `GMSACredentialSpec CRD`
+
+Uma [CustomResourcedefinition](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)(CRD) uma especificação de recursos de credencial GMSA precisa ser configurada no cluster, para definir o tipo de recurso do cliente `GMSACredentialSpec`. Faça o download da GMSA CRD [YAML](https://github.com/kubernetes-sigs/windows-gmsa/blob/master/admission-webhook/deploy/gmsa-crd.yml) 
+e salve como gmsa-crd.yaml.
+A seguir, instale o CRD com `kubectl apply -f gmsa-crd.yaml`
+
+### Instale `webhooks` para validar usuários GMSA
+
+Dois `webhooks` precisam ser configurados no cluster Kubernetes para popular e validar 
+as referências de especificação de credenciais GMSA no nível do Pod ou contêiner:
+
+1. Um webhook mutante que expanda as referências para as GMSAs,
+(por nome de uma especificação de Pod) dentro de toda a especificação de credencial, 
+no formato JSON dentro da especificação do Pod.
+
+1. Um webhook de validação garante que todas as referências para GMSAs estão 
+autorizadas a serem usadas pela conta de serviço do Pod.
+
+A instalação dos webhooks acima e dos objetos associados requer as etapas abaixo: 
+
+1. Crie um par de chaves de certificado (que será usado para permitir que o 
+contêiner do webhook se comunique com o cluster)
+
+1. Instale a `secret` com o certificado acima.
+
+1. Crie uma implantação para `core webhook logic`.
+
+1. Crie as configurações de validação e o webhook mutante, referentes à implantação.
```

Comment:
This sentence is not quite correct. In the original docs, the meaning is to point the reader to create configurations for both validating and mutating webhooks that will reference the Deployment created in the previous step. I can't quite grasp that meaning from this sentence.